src/components/network/geo-data.js

```diff
@@ -115,6 +115,7 @@ export default class GeoData {
             ];
         }
 
+        // why it's not at the start of the function ?
```

Comment:
Yes it can be refacto with the test at the start

Reply:
Ok I'll do it in an other PR